### ISSUE TITLE
Question: Image Builder and Conda

### ISSUE THREAD

**Issue by sherl0cks** [jupyter-on-openshift/jupyter-notebook-builder#1]
Why does the image builder rely on pip to install packages / kernels when the python environment is set to conda? For context, I have a customer looking to work from conda for both python 3 and R. I'd like to collaborate on s2i notebook builders to met there goals. There are a number of repos in this org, so let me know if I'm looking in the wrong place.

https://github.com/jupyter-on-openshift/jupyter-image-builder/blob/master/image-builder/assemble#L158

**Comment by GrahamDumpleton** [comment]
Note that this repository is being superseded by https://github.com/jupyter-on-openshift/s2i-notebook-composite

In the first pass I am using ``pip`` and only honouring ``requirements.txt`` as it is easier and works no matter what Python distribution you are using.

Supporting ``conda`` for installing packages from the separate Anaconda package repository, in addition to supporting ``conda`` and ``environment.yml`` gets more complicated. You also have to contend with how you identify different sets of packages that may need to be installed if running multiple Python versions in different kernels. The desire to use a separate ``conda`` environment vs installing into default environment can be an issue. So lots of things and taking it a step at a time.

**Comment by sherl0cks** [comment]
makes sense about the repo being superseded, though a `README` that says so would be good so folks are aware.

also, first pass with just `pip` makes sense.

RE: `conda` and the multiple env and channel concern, I've taken a stab with https://github.com/sherl0cks/anaconda-s2i/blob/master/conda-jupyter-base/s2i/bin/assemble. example can be seen https://github.com/sherl0cks/anaconda-s2i/tree/master/jupyter-kitchen-sink.

**Comment by GrahamDumpleton** [comment]
That there was no ``README`` at all to begin with is a good sign that either way, it was a work in progress. ;-)

**Comment by sherl0cks** [comment]
Yeap, it was a sign indeed. Just a thought to help those exploring this repo in the future, not a criticism of what was done in the past. Happy to send a quick PR tomorrow if helpful (I'm in Denver - UTC-7)

**Comment by GrahamDumpleton** [comment]
No need. I should be getting back into working on this stuff tomorrow. Got put aside while I was on holidays for last couple of weeks, which is why didn't finish cleaning up the composite builder.
